extensions/podman/packages/extension/src/podman5.json

```diff
@@ -6,7 +6,14 @@
   "platform": {
     "win32": {
       "version": "v5.6.0",
-      "fileName": "podman-5.6.0-setup.exe"
+      "arch": {
+        "x64": {
+          "fileName": "podman-5.6.0-setup.exe"
```

Comment:
I suggest you use podman-installer-windows-amd64.exe instead (its the exact same binary), the other location has been "deprecated".